examples/example_tissuemask.ipynb

```diff
@@ -0,0 +1,410 @@
+{
```

Comment:
<p>a standard browser <strong>(preferably latest version of Google Chrome)</strong> and no programming language <strong>installation </strong>is</p><p> </p><p>It can run <strong>be </strong>remotely over the Internet</p><p> </p><p>Please link README to the master branch</p><p> </p><p>also link details to master branch</p><p> </p><p>Link Otsu to some book or opencv package</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Isn't the Wikipedia page better for referencing in this case? because it is a web example and it is OK to cite Wikipedia. Also, the Wikipedia is somehow more general, has some images, and easily accessible in comparison to books or CV documentations. But if you are happy with another reference, I'm happy to replace it. </p>

Reply:
<p>That's odd, I applied these comments the first thing last time. Somewhere in the process they were not applied. Anyway, all included now. Thanks</p>